src/main/java/seedu/address/ui/Tabs.java

```diff
@@ -0,0 +1,160 @@
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+public class Tabs extends VBox {
+
+    //FXML
+    private static final String FXML = "/view/Tabs.fxml";
+
+    //FXML properties
+    private static final String TRANSPARENT = "-fx-background-color: transparent;";
+    private static final String TAB_COLOR = "-fx-background-color: #BED0F7;";
+    private static final String INTERNSHIP_Y_TRANSLATE = "-fx-translate-y: 0";
+    private static final String COMPANY_Y_TRANSLATE = "-fx-translate-y: -13.5";
+    private static final String USER_Y_TRANSLATE = "-fx-translate-y: -27";
+
+    // Image links
+    private static final String INTERNSHIP_IMAGE_LINK = "/images/internship.png";
+    private static final String COMPANY_IMAGE_LINK = "/images/company.png";
+    private static final String USER_IMAGE_LINK = "/images/user.png";
+
+    @FXML
+    private Button internshipButton;
+    @FXML
+    private Button companyButton;
+    @FXML
+    private Button userButton;
+    @FXML
+    private VBox internship;
+    @FXML
+    private VBox company;
+    @FXML
+    private VBox user;
+    @FXML
+    private ImageView internshipIcon;
+    @FXML
+    private ImageView companyIcon;
+    @FXML
+    private ImageView userIcon;
+
+    private Stage stage;
+    private MainWindow mainWindow;
+
+    /**
+     * Constructs the {@code Tabs} in the given {@code primaryStage} of the {@code mainWindow}.
+     */
+    private Tabs(MainWindow mainWindow, Stage primaryStage) {
+        try {
+            stage = primaryStage;
+            this.mainWindow = mainWindow;
+            // Loading the information of the GUI
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource(FXML));
+            fxmlLoader.setController(this);
+            fxmlLoader.setRoot(this);
+            fxmlLoader.load();
+            internship.setStyle(TAB_COLOR);
+            // initialize the GUI of the tabs.
+            setTabIcons();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Creates the {@code Tabs} information in the {@code primaryStage} of the {@code mainWindow}.
+     */
+    public static Tabs getTabs(MainWindow mainWindow, Stage primaryStage) {
+        return new Tabs(mainWindow, primaryStage);
+    }
+
+    /**
+     * Handles the click {@code event} on the tabs.
+     */
+    @FXML
+    public void handleClick(ActionEvent event) throws IOException {
+        if (event.getSource() == internshipButton) {
+            selectInternship(stage);
+        } else if (event.getSource() == companyButton) {
+            selectCompany(stage);
+        } else if (event.getSource() == userButton) {
+            selectUser(stage);
+        } else {
+            assert false : "Invalid button";
```

Comment:
Perhaps its more appropriate a throw an exception here since assertions are mainly to check for precondition and post condition of function execution and testing assumptions. This usage fits neither and is more of an exceptional case?